### DISCUSSION TITLE
Access denied - This application can not run on your computer

### DISCUSSION THREAD

**Discussion by migueleardo** [NVlabs/instant-ngp#535] [Q&A]
Hi,

I went through whole installation process successfully (at least it seemed so). When I try to run testbed, I get this error saying the program can not be run on my computer. Thanks in forward for your help
![image](https://user-images.githubusercontent.com/59333496/165252402-8952c3ce-6773-4a78-9085-827b7227ead9.png)

**Comment by N0ah10f1** [comment]
Hi, have you solved this problem?
